#!/usr/bin/env python
PACKAGE = "simple_filter"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sigma_m_sq",    double_t,    0, "Variance of the motion noise", .01,  0., 1.)
gen.add("sigma_z_sq", double_t, 0, "Variance of the measurement noise",    .1, .00001,  1)
gen.add("pause_time",    double_t,    0, "The delay between plots",  1.0, 0.1, 2.0)

exit(gen.generate(PACKAGE, "simple_filter", "SimpleKalman"))